app/views/layouts/_navbar.html.erb

```diff
@@ -1,56 +1,43 @@
-<nav class="navbar navbar-default navbar-fixed-top" role="navigation">
-  <div class="container-fluid">
-    <div class="row ui-shescoding-nav <%= add_custom_nav_class %>">    
-      <div class="nav-header">  
-        <div class="col-sm-4">
-        
-          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
-            data-target="#example-navbar-collapse">
-            <span class="sr-only">Toggle navigation</span>
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-            <div class="collapse navbar-collapse" id="example-navbar-collapse">
-              <ul>
-                <li><%= link_to "About", "/about" %></li>
-                <li><%= link_to "Resources", "/resources" %></li>
-                <li><%= link_to "Team", "/team" %></li>
-                <li><%= link_to "Connect", "/contact" %></li>
-              </ul>
-            </div>
-          </button>
-          <a class="navbar-brand" href="/">
-            <%= image_tag('Logo-White_HighRes.png') %>
-          </a>
-        </div>
-      </div>
-      <div class="col-sm-7  col-sm-offset-1">
-        <ul class="nav navbar-nav navbar-right">
-          <% if page == "About" %>
-            <li class="active"><%= link_to "About", "/about" %></li>
-          <% else %>
-            <li><%= link_to "About", "/about" %></li>
-          <% end %>
+<nav class="navbar navbar-default navbar-fixed-top ui-shescoding-nav <%= add_custom_nav_class %>" role="navigation">
+  <div class="container">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
+          data-target="#navbar-links">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand" href="/">
+        <%= image_tag('Logo-navbar.png') %>
+      </a>
+    </div>
+    <div class="collapse navbar-collapse" id="navbar-links">
+      <ul class="nav navbar-nav">
```

Comment:
We'll just need to add the navbar-right back in, such that the menu items are aligned on the right. It still works nicely when decreasing to XSmall.